BUILD.md

```diff
@@ -99,22 +99,25 @@ linkerd2_components
 Depending on use case, there are several configurations with which to develop
 and run Linkerd2:
 
-- [Comprehensive](#comprehensive): Integrated configuration using Minikube, most
+- [Comprehensive](#comprehensive): Integrated configuration using kubernetes for Docker, most
   closely matches release.
 - [Web](#web): Development of the Linkerd2 Dashboard.
 
 ### Comprehensive
 
 This configuration builds all Linkerd2 components in Docker images, and deploys
-them onto Minikube. This setup most closely parallels our recommended production
+them onto local kubernetes cluster. This setup most closely parallels our recommended production
 installation, documented at https://linkerd.io/2/getting-started/.
 
 These commands assume a working
```

Comment:
Mind just linking to the k8s cluster install docs? That way we're not picking a specific one.

Reply:
Sorry for the delay in getting back to this. 
I am not 100% sure about what you are referring to by `k8s cluster install docs`?

Reply:
https://kubernetes.io/docs/setup/